backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/validator/storage/MultipleStorageDomainsValidator.java

```diff
@@ -161,6 +161,15 @@ public ValidationResult allDomainsHaveSpaceForDisksWithSnapshots(Collection<Disk
         });
     }
 
+    /**
+     * Validates that all target domains have enough space for the copied disks.
+     * @return {@link ValidationResult#VALID} if all the domains have enough free space, or a {@link ValidationResult}
+     * with the first low-on-space domain encountered.
+     */
+    public ValidationResult allTargetDomainsHaveSpaceForDisks(Collection<DiskImage> diskImages) {
+        return validOrFirstFailure(entry -> getStorageDomainValidator(entry).hasSpaceForDisksWithSnapshots(diskImages));
```

Comment:
this puzzles me a bit - the method's name suggests that all domin**s** are checked to see that they can cope the creation of the specified disk but there it appears as the precondition here is that unlike allDomainsHaveSpaceForDisksWithSnapshots this assumes the 'entries' contain a single target domain that all the specified disks are going to be copied to, is that right? if that's correct, how about renaming it to targetDomainHasSpaceForDisks?

Reply:
yes, allTargetDomainHaveSpaceForDisks makes more sense